pmm-app/src/pmm-dbaas/components/DBCluster/OptionContent/OptionContent.styles.ts

```diff
@@ -0,0 +1,44 @@
+import { css } from 'emotion';
+
+export const styles = {
+  optionWrapper: css`
+    align-items: center;
+    display: flex;
+    justify-content: space-between;
+  `,
+  optionText: css`
+    display: flex;
+    flex-direction: column;
+    max-width: 75%;
+    white-space: normal;
+  `,
+  optionTitle: css`
+    color: #d8d9da;
```

Comment:
Aren't there any theme colors that can be used here?

Reply:
Used styles exactly the same we used in QAN, good point that it would be better to refactor it so created a separate task for it: https://jira.percona.com/browse/PMM-7205